scripts/migrate_to_rapidpro/fix_auto_baby_switch_data.py

```diff
@@ -0,0 +1,259 @@
+import asyncio
+import csv
+import os
+import time
+from datetime import datetime, timedelta
+
+import aiohttp
+import psycopg2
+from six.moves import urllib_parse
+
+CONCURRENCY = 10
+HUB_OUTPUT_FILE = "hub_babyswitches.csv"
+RAPIDPRO_OUTPUT_FILE = "rapidpro_babyswitch_updates.csv"
+
+LIMIT = 10_000_000
+RAPIDPRO_URL = "https://rapidpro.prd.momconnect.co.za/"
+RAPIDPRO_TOKEN = os.environ["RAPIDPRO_TOKEN"]
+
+HUB_DB_PASSWORD = os.environ["HUB_PASS"]
+
+total = 0
+excluded = 0
+start, d_print = time.time(), time.time()
+
+
+async def get_rapidpro_contact(session, contact_id):
+    url = urllib_parse.urljoin(RAPIDPRO_URL, f"/api/v2/contacts.json?uuid={contact_id}")
+    headers = {
+        "Authorization": f"TOKEN {RAPIDPRO_TOKEN}",
+        "Content-Type": "application/json",
+        "Connection": "Keep-Alive",
+    }
+    async with session.get(url, headers=headers) as response:
+        response_body = await response.json()
+
+        if response_body["results"]:
+            return response_body["results"][0]
+
+        return None
+
+
+def in_postbirth_group(contact):
+    for group in contact["groups"]:
+        if "post" in group["name"].lower():
+            return True
+
+    return False
+
+
+def get_contact_msisdn(contact):
+    for urn in contact["urns"]:
+        if "whatsapp" in urn:
+            return "+" + urn.split(":")[1]
+
+
+def get_baby_dob_field(fields):
+    for i in range(1, 4):
+        dob_field = f"baby_dob{i}"
+        if not fields[dob_field]:
+            return dob_field
+
+
+def get_babyswitches(conn):
+    babyswitches = {}
+    cursor = conn.cursor("baby_switches")
+    print("Fetching Baby Switches...")
+    cursor.execute(
+        f"""
+        select contact_id, timestamp
+        from eventstore_babyswitch
+        order by timestamp desc
+        limit {LIMIT}
+        """
+    )  # 158680
+    total = 0
+    start, d_print = time.time(), time.time()
+    for (contact_id, timestamp) in cursor:
+        babyswitches[contact_id] = {"timestamp": timestamp}
```

Comment:
We can probably save a bunch of memory by just storing `timestamp`, instead of a dictionary with the timestamp in it, I can't see anywhere where we would require this to be a dict